### ISSUE TITLE
Nested rules order mess up

### ISSUE THREAD

**Issue by changbenny** [MoOx/postcss-cssnext#258]
I'm trying to use the nest features in cssnext, but the below style
```
.happy {
    & 1 {
        position: relative;
    }
    & 2 {
        position: relative;
    }
}
```
will produce the following result:
```
.happy 2 {
    position: relative;
}
.happy 1 {
    position: relative;
}
```
The order is reversed. Is there any suggestions? Thanks.

**Comment by MoOx** [comment]
Indeed that's weird. Should be an issue for https://github.com/jonathantneal/postcss-nesting poke @jonathantneal

**Comment by jonathantneal** [comment]
Yes, I am resolving the non-`@nest` version of nesting. It only happens when you do not include `@nest`.

**Comment by jonathantneal** [comment]
This has been resolved in [v2.2.0](https://github.com/jonathantneal/postcss-nesting/blob/v2.2.0/CHANGELOG.md).

**Comment by MoOx** [comment]
Ok cool!
So people affected by this bug, just uninstall and reinstall postcss-cssnext to fix the issue.
